src/quo2/components/tags/tags.cljs

```diff
@@ -26,4 +32,121 @@
                                       resource)
                      :on-press      #(do (reset! active-tab-id %)
                                          (when on-change (on-change %)))})
-             tag-label]])]))))
\ No newline at end of file
+             tag-label]])]))))
+
+(defn- calculate-fade-end-percentage
```

Comment:
is it a copy from tabs? might be worth just moving it and use in both places instead of copying it?

Reply:
I second Andrey on this ..